### PR TITLE
Create aad-sales newsletters

### PR DESCRIPTION
**Pull request by jwade1327**
Screen Shot 2023-11-21 at 7 32 03 AM

### CODE REVIEW

**Comment by jwade1327** [comment]
[New eNewsletter Template for AAD Sales](https://trello.com/c/cnZBLOg0/520-new-enewsletter-template-for-aad-sales)

File: packages/common/components/blocks/footer-aad-sales.marko [hunk 1]
```diff
@@ -0,0 +1,149 @@
+import { get, getAsObject, getAsArray } from "@parameter1/base-cms-object-path";
+import moment from "moment";
+
+$ const { website, config } = out.global;
+$ const { newsletter, date } = input;
```
Comment (Shinsina) [inline packages/common/components/blocks/footer-aad-sales.marko:5]: `getAsObject`, `moment` and `date` go unused here

File: packages/common/components/layouts/sales.marko [hunk 2]
```diff
@@ -0,0 +1,177 @@
+import { get } from "@parameter1/base-cms-object-path";
+import { parseBooleanHeader } from "@parameter1/base-cms-utils";
+import queryFragment from "@ascend-media/package-common/graphql/fragments/content-list";
+
+$ const { website, config, req } = out.global;
+$ const platformDesignator = 'd';
+$ const { newsletter, date } = input.data;
```
Comment (Shinsina) [inline packages/common/components/layouts/sales.marko:7]: `parseBooleanHeader`, `queryFragment`, `website`, and `req` go unused here.

**Review by Shinsina — CHANGES_REQUESTED** [review]
Just some unused stuff we can prune out here. Let me know if there are any questions or issues here.

**Review by Shinsina — APPROVED** [review]

**Pull request merged**